File: e2e/fixtures/preview-matrix/code/sample.go
// Beebeeb preview-matrix fixture — Go (task 1565).
package main

import "fmt"

const greeting = "café naïve 日本語 Ελληνικά Москва €19.99"

func main() {
	longLine := ""
	for i := 0; i < 20; i++ {
		longLine += "0123456789 "
	}
	longLine += "end-of-long-line"
	fmt.Println(greeting)
	fmt.Println(longLine)
}

File: e2e/fixtures/preview-matrix/code/sample.swift
// Beebeeb preview-matrix fixture — Swift (task 1565).

let greeting = "café naïve 日本語 Ελληνικά Москва €19.99"

func longLine() -> String {
    var parts = [String]()
    for _ in 0..<20 { parts.append("0123456789") }
    return parts.joined(separator: " ") + " end-of-long-line"
}

print(greeting)
print(longLine())
